workflow/scripts/generate_blacklist.py

```diff
@@ -0,0 +1,49 @@
+import os
+import re
+import yaml
+from smart_open import open
+
+# download blacklist and trim it for a specific chromosome
+
+
+def copy_blacklist(igenomes_or_blacklist, blacklist_path):
+    with open(igenomes_or_blacklist) as fin:
+        with open(blacklist_path, 'w') as fout:
+            for line in fin:
+                fout.write(line)
+
+
+def get_blacklist_from_igenomes(igenomes_or_blacklist, blacklist_path):
+    with open(igenomes_or_blacklist) as f:
+        igenomes = yaml.load(f, Loader=yaml.FullLoader)
+        if "blacklist" in igenomes["params"]["genomes"][build]:
+            blacklist_link = igenomes["params"]["genomes"][build]["blacklist"]
+            with open(blacklist_link) as fin:
+                with open(blacklist_path, 'w') as fout:
+                    pattern = re.compile(r'^chr')
+                    for line in fin:
+                        formatted_line = re.sub(pattern, "", line)
```

Comment:
I don't think the `re` module import is needed for this:
```suggestion
                    for line in fin:
                        if line.startswith("chr"):
                            formatted_line = line.replace("chr","",1)
                        else formatted_line = line
```

Maybe even shorter, if we simply go for `line` instead of `formatted_line` in the following, as this will eliminated the else clause. Also, the `import re` can then go.